sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/pubsub/PubsubIO.java

```diff
@@ -598,274 +634,212 @@ private Read(String name, ValueProvider<PubsubSubscription> subscription,
      *
      * @see <a href="https://www.ietf.org/rfc/rfc3339.txt">RFC 3339</a>
      */
-    public Read<T> timestampLabel(String timestampLabel) {
-      return new Read<>(
-          name, subscription, topic, timestampLabel, coder, idLabel,
-          parseFn);
+    public Read<T> withTimestampLabel(String timestampLabel) {
```

Comment:
"label" is a remnant of the old beta PubSub API. I believe PubSub now calls these attributes instead - maybe we should rename here to be withTimestampAttribute? (and withIdAttribute)